ios/RNSScreenStack.mm

```diff
@@ -747,6 +742,31 @@ - (UIViewController *)reactViewController
   return _controller;
 }
 
+- (BOOL)isInGestureResponseDistance:(UIGestureRecognizer *)gestureRecognizer topScreen:(RNSScreenView *)topScreen
+{
+  NSDictionary *gestureResponseDistanceValues = topScreen.gestureResponseDistance;
+  float x = [gestureRecognizer locationInView:gestureRecognizer.view].x;
+  float y = [gestureRecognizer locationInView:gestureRecognizer.view].y;
+  BOOL isRTL = _controller.view.semanticContentAttribute == UISemanticContentAttributeForceRightToLeft;
+  if (isRTL) {
+    x = _controller.view.frame.size.width - x;
+  }
+  
+  // see: https://github.com/software-mansion/react-native-screens/pull/1442/commits/c0413bddadc1023022d3d390373321632ad3539d
```

Comment:
This commit is not available for me.

Reply:
Fixed it here: [ed66a55](https://github.com/software-mansion/react-native-screens/pull/1446/commits/ed66a55157243dc2d7930480589e25a70d00063e)